### DISCUSSION TITLE
Enhancement: Add greek letter version of constants

### DISCUSSION THREAD

**Discussion by orelbn** [josdejong/mathjs#3335] [Ideas]
Currently, pi can be evaluated as constant:

![image](https://github.com/user-attachments/assets/b7ca34e3-5b99-4f7b-9bc3-fa04bc9cc236)

But the Greek version can not be:

![image](https://github.com/user-attachments/assets/0115276a-386b-4821-ba9a-e08c073a648b)

It's the same situation for phi.

It would be nice to be able to use the Greek letter equivalent as constants as well, without needing to set it explicitly.

![image](https://github.com/user-attachments/assets/f0cd055e-499f-4574-ba65-9e182a4e0311)

**Comment by josdejong** [comment]
The library restricts itself to ascii characters, though it allows using non-ascii characters like π. But indeed, you can load aliases yourself into the library so there is no limitation in that regard.
